src/widgets/Finance/components/Finance.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+const Finance = ({ finance }) =>
+  <div className="panel Number">
+    <div className="panel-heading">Rocket Internet SE Stock Price</div>
+    <div className="panel-body">
+      <span className="current">
+        {finance.finance.l_fix}
```

Comment:
We should re-package the stock data on the server side before returning it instead of sending it over raw. That way we can add some post processing logic later on the server side.